apps/climatemappedafrica/src/components/Header/Header.js

```diff
@@ -50,16 +52,26 @@ const Header = React.forwardRef(function Header(props, ref) {
       >
         {children}
       </RichTypography>
-      <RichTypography
-        variant="subtitle1"
-        {...SubtitleProps}
-        sx={{
-          mt: 20,
-          ...SubtitleProps?.sx,
-        }}
-      >
-        {subtitle}
-      </RichTypography>
+      {typeof subtitle === "string" ? (
+        <RichTypography
```

Comment:
Cool but lets leave this header as is (just string)... I've created a [`RichHeader`](https://github.com/CodeForAfrica/ui/pull/952/files#diff-ab220b51a3db21b1094cd4fbc66d9b4b821bdb22fe9059db62bcb38881c21228) component that handles rich text.